### PR TITLE
[12.0][ADD] account_invoice_report_hide_line

### PR DESCRIPTION
**Pull request by bodedra**
Issue Reference: https://github.com/OCA/account-invoice-reporting/issues/114

CC @max3903

### CODE REVIEW

File: account_invoice_report_hide_line/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import account_invoice
```
Comment (max3903) [inline account_invoice_report_hide_line/models/__init__.py]: @bodedra Missing newline.

Can you also change the copyright header to:
```
# Copyright (C) 2019 Open Source Integrators
```

Thanks!

File: account_invoice_report_hide_line/views/account_invoice_view.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (max3903) [inline account_invoice_report_hide_line/views/account_invoice_view.xml]: I think this line can be removed.

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by bodedra** [comment]
@max3903 Comment attended! Thanks for your review. :+1:

**Review by max3903 — APPROVED** [review]

**Comment by max3903** [comment]
@RLeeOSI Can you review this pull request?

**Review by osi-scampbell — APPROVED** [review]

**Comment by max3903** [comment]
@bodedra Can you add unit tests to maintain/increase the code coverage?

**Comment by RLeeOSI** [comment]
Checked by Raphael Lee. No conflicts found. Should this capability be added to quotations as well?

**Comment by max3903** [comment]
@RLeeOSI Nope.

**Comment by bodedra** [comment]
> @bodedra Can you add unit tests to maintain/increase the code coverage?

@max3903 Unit tests added.

**Pull request merged**